### PR TITLE
hopefully this will fix the bug

### PR DESCRIPTION
**Pull request by olech2412**

### CODE REVIEW

**Comment by olech2412** [comment]
@whosFritz please pull this branch and check your features (espacially the trainstatistics) if everything makes sense locally with a new up to date database

**Comment by olech2412** [comment]
references: https://github.com/whosFritz/Rail-Insights/issues/93

File: src/main/java/de/whosfritz/railinsights/calculation/UniversalCalculator.java [hunk 1]
```diff
@@ -356,7 +356,7 @@ public TripCounts countTrips(List<Trip> trips, LocalDateTime from, LocalDateTime
             }
         }
 
-        for(LocalDateTime date : hourlyTripCounts.keySet()) {
+        for (LocalDateTime date : hourlyTripCounts.keySet()) {
```
Comment (whosFritz) [inline src/main/java/de/whosfritz/railinsights/calculation/UniversalCalculator.java:359]: bloat

File: src/test/java/de/whosfritz/railinsights/utils/PercentageUtilTest.java
Comment (whosFritz) [inline src/test/java/de/whosfritz/railinsights/utils/PercentageUtilTest.java:1]: keine tests, tests sind bloat

**Review by whosFritz — APPROVED** [review]

**Pull request merged**